installer/scripts/omsadmin.sh

```diff
@@ -310,6 +313,36 @@ set_proxy_setting()
     fi
 }
 
+onboard_scom()
+{
+    echo "Onboarding SCOM"
+    create_workspace_directories "scom"
+
+    if [ ! -f $CERT_DIR/scom-cert.pem ] || [ ! -f $CERT_DIR/scom-key.pem ]; then
+        echo "generating SCOM certs"
+        # Generate client auth cert using scxsslconfig tool.
+        $SCX_SSL_CONFIG -c -g $CERT_DIR
+        if [ $? -ne 0 ]; then
+          log_error "Error generating certs"
+          clean_exit 1
+        fi
+        # Rename cert/key to more meaningful name
+        mv $CERT_DIR/omi-host-*.pem $CERT_DIR/scom-cert.pem
+        mv $CERT_DIR/omikey.pem $CERT_DIR/scom-key.pem
+        rm -f $CERT_DIR/omi.pem
+        chown_omsagent $CERT_DIR/scom-cert.pem
```

Comment:
Is 333 and 334 necessary? Oh, I guess if you're running as root, they are. But not if you're running as omsagent. But in that case, it's harmless.